models/Model.js

```diff
@@ -47,4 +57,30 @@ exports.DatabaseHandler = class {
     // delete the doggo
     await collection.deleteOne({ userId: bodyId });
   }
+
+  async editProfile(req) {
+    const database = this.client.db("DoggoSwipe");
+    const collection = database.collection("Users");
+
+    let Id = req.body.userId;
+    // https://stackoverflow.com/questions/8233014/how-do-i-search-for-an-object-by-its-objectid-in-the-mongo-console
+    let o_id = new ObjectId(Id);
+    let data = req.body;
+
+    for (const item in data) {
+      let update = `${data[item]}`;
+      // console.log(item);
+      if (item === "breed") {
```

Comment:
Dit gaat niet helemaal goed, in de database moet het een array zijn met aparte strings, maar nu zie ik een array met 1 string en daarin de breeds.

Reply:
Jaa ik zie het nu. Ik ga nog even een nieuwe branch aanmaken vanuit dev en fix het voor je :)

Reply:
Top!